landing/landing.js

```diff
@@ -13,6 +13,16 @@ var previewFrame = document.getElementById("preview-frame");
 var getCodeEl = document.getElementById("getcode");
 var embedCodeEl = document.getElementById("embed-code");
 
+var copyCodeToClipboard = function() {
```

Comment:
Maybe moving this function to a more relevant place ?

Reply:
I moved it to the end of the functions. Maybe that makes more sense.

Reply:
Maybe below `endGetCode` instead ?

Reply:
https://github.com/Andreto/scratch-profile-embed/commit/c55fbc75b8a1d105685249175957d7d94686d46a